package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "node ./app/app.js",
     "lint": "eslint ./app/*/*.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "up": "docker-compose up -d --build",
+    "down": "docker-compose down",
+    "validate": "./devops/smart-install.sh"
```

Comment:
Why do we validate, did we not agree that --build solve all the problems?

Reply:
I was unsure if the behaviour met the criteria for the linked ticket. Do you want me to remove it?

Reply:
I'd say so. You can remove validate and  smart-install, and consider ticket to be closed after mergint this PR. I'll approve it.